### PR TITLE
White space changes

### PR DESCRIPTION
**Pull request by ikegami-t**
Includes some minor changes also. Just split the chagnes from the PR #2188 but deleted the long line string changes.

### CODE REVIEW

**Comment by ikegami-t** [comment]
Todo: Change register value output to use the macros `NVME_PMRCAP_CMSS` and other `NVME_***_***`.

File: nvme-print-stdout.c [hunk 1]
```diff
@@ -3264,20 +3278,21 @@ static void stdout_zns_changed(struct nvme_zns_changed_zone_log *log)
 
 static void stdout_zns_report_zone_attributes(__u8 za, __u8 zai)
 {
-	const char *const recommended_limit[4] = {"","1","2","3"};
+	const char *const recommended_limit[4] = {"", "1", "2", "3"};
```
Comment (calebsander) [inline nvme-print-stdout.c:3281]: Add a space between `*` and `const` while you're touching this line? `const char * const`

**Review by calebsander — COMMENTED** [review]
Generally looks good!

File: nvme-print-stdout.c (hunk 1 above)
Comment (ikegami-t) [reply nvme-print-stdout.c]: Yes done it.

File: Documentation/nvme-set-property.txt [hunk 2]
```diff
@@ -23,8 +23,8 @@ OPTIONS
 --offset::
 	The offset of the property.
 
--V::
---value:
+-V <val>::
+--value=<val>::
```
[the remark this answers is not in the document]
Comment (ikegami-t) [reply Documentation/nvme-set-property.txt]: Missed it so just fixed.

**Review by calebsander — APPROVED** [review]

File: nvme-print-stdout.c [hunk 3]
```diff
@@ -996,20 +992,20 @@ static void stdout_supported_cap_config_log(struct nvme_supported_cap_config_lis
...
 			printf("Number of NVM Sets: %u\n", egsets);
-			for(k = 0; k < egsets; k++) {
+			for (k = 0; k < egsets; k++) {
 				printf("NVM Set %d Identifier: %u\n", i,
 					le16_to_cpu(cap->cap_config_desc[i].egcd[j].nvmsetid[k]));
 			}
-			chan_desc = (struct nvme_end_grp_chan_desc *) \
-					((cap->cap_config_desc[i].egcd[j].nvmsetid[0]) * (sizeof(__u16)*egsets));
+			chan_desc = (struct nvme_end_grp_chan_desc *)
+			    (sizeof(__u16) * egsets * cap->cap_config_desc[i].egcd[j].nvmsetid[0]);
```
Comment (igaw) [inline nvme-print-stdout.c]: This part should be in the next patch. It's not a white space change thus it should get a proper commit message with it.

File: nvme-print-stdout.c [hunk 4]
```diff
@@ -4635,7 +4648,9 @@ static void stdout_generic_full_path(nvme_ns_t n, char *path, size_t len)
 	int instance;
 	struct stat st;
 
-	sscanf(nvme_ns_get_name(n), "nvme%dn%d", &instance, &head_instance);
+	if (sscanf(nvme_ns_get_name(n), "nvme%dn%d", &instance, &head_instance) != 2)
+		return;
+
```
Comment (igaw) [inline nvme-print-stdout.c:4653]: Not a whitespace change. Should be in a different patch.

File: nvme-print-stdout.c [hunk 5]
```diff
@@ -992,12 +992,12 @@ static void stdout_supported_cap_config_log(struct nvme_supported_cap_config_lis
 					cap->cap_config_desc[i].egcd[j].end_est)));
 			egsets = le16_to_cpu(cap->cap_config_desc[i].egcd[j].egsets);
 			printf("Number of NVM Sets: %u\n", egsets);
-			for (k = 0; k < egsets; k++) {
+			for (k = 0; k < egsets; k++)
 				printf("NVM Set %d Identifier: %u\n", i,
-					le16_to_cpu(cap->cap_config_desc[i].egcd[j].nvmsetid[k]));
-			}
+				       le16_to_cpu(cap->cap_config_desc[i].egcd[j].nvmsetid[k]));
+
```
Comment (igaw) [inline nvme-print-stdout.c:998]: Split whitespace change and logical changes into separate patches.

File: Documentation/nvme-get-property.txt [hunk 6]
```diff
@@ -19,8 +19,8 @@ Reads and shows the defined NVMe controller property for NVMe over Fabric.
 
 OPTIONS
 -------
--O::
---offset::
+-O <offset>::
+--offset=<offset>::
```
Comment (igaw) [inline Documentation/nvme-get-property.txt:23]: Just split the two different changes into two patches. One which fixes the mode settings and one which fixes the offset changes. Thanks.

**Comment by igaw** [comment]
A few nitpicks. Could you rebase the series to the latest upstream version when you updating the series? There is merge conflict in nvme-print.c Thanks.
